src/Modules/Backend/Domain/ModuleSettings/ModuleSettingsType.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace ForkCMS\Modules\Backend\Domain\ModuleSettings;
+
+use ForkCMS\Core\Domain\Form\SwitchType;
+use ForkCMS\Modules\Extensions\Domain\Module\Command\ChangeModuleSettings;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Expression;
+
+final class ModuleSettingsType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add(
+                '2fa_enabled',
+                CheckboxType::class,
+                [
+                    'required' => false,
+                    'label' => 'lbl.EnableTwoFactorAuthentication',
+                    'label_attr' => ['class' => 'checkbox-switch'],
+                ]
+            )
+            ->add(
+                '2fa_key',
+                TextType::class,
+                [
+                    'label' => 'lbl.TwoFactorAuthenticationKey',
+                    'required' => false,
+                ]
+            )
+            ->add(
+                '2fa_required',
+                CheckboxType::class,
+                [
+                    'required' => false,
+                    'label' => 'lbl.RequireTwoFactorAuthentication',
+                    'label_attr' => ['class' => 'checkbox-switch'],
+                ]
+            )
+            ->add(
+                'trusted_devices_enabled',
+                CheckboxType::class,
+                [
+                    'required' => false,
+                    'label' => 'lbl.EnableTrustedDevices',
```

Comment:
Seems useful to add some help text that explains what this does. eg `When enabled users will be allowed to trust a device for 30 days. On a trusted device the verification code will not be asked.`

Reply:
fixed 5b995b91ede947614eb480c3a83337354604c688